docs/maatregelen/2-owp-16-leveren-bewijs-door-aanbieder-niet-schenden-auteursrechten.md

```diff
@@ -0,0 +1,70 @@
+---
+# vul hier een titel in voor deze maatregel
+title: Laat aanbieder(s) bewijs leveren dat de door hen ontwikkelde algoritmes geen inbreuk maken op de auteursrechten van derden met de trainingsdata en de output.
+id: urn:nl:ak:mtr:owp-16
+# geef hier een korte toelichting van deze maatregel
+toelichting: Maak het al dan niet kunnen leveren van bewijs door een aanbieder dat auteursrechten niet worden geschonden door de trainingsdata of out van een algoritme een vast onderdeel van de aanbesteding.
+# vul hier de bestandsnamen in van de vereisten die horen bij deze maatregel
+vereiste:
+- aut-01-auteursrechten
+  
+# vul hier de fasen van de levenscyclus in die horen bij deze maatregel
+- ontwerp
+
+# vul hier de bouwblokken in die horen bij deze maatregel
+- publieke-inkoop
+hide:
+- navigation
+  
+# geef hier aan welke rollen bij deze maatregelen betrokken kunnen zijn
+- projectleider
+- beleid-en-advies 
+
+hide:
+- navigation
+- toc
+---
+
+<!-- Let op! onderstaande regel met 'tags' niet weghalen! Deze maakt automatisch de knopjes op basis van de metadata  -->
+<!-- tags -->
+
+## Maatregel
+
+Maak het al dan niet kunnen leveren van bewijs door een aanbieder dat auteursrechten niet worden geschonden met de trainingsdata en output van diens algoritme van bijvoorbeeld een aanbesteding.
+
+## Toelichting
+
+### Trainingsdata
+- Algoritmes worden veelal getraind aan de hand van een omvangrijke hoeveelheid data. Wanneer grote hoeveelheden data, bijvoorbeeld door deze te scrapen van internet, worden gebruikt is het zeer aannemelijk (of: nagenoeg zeker) dat zich onder de gescrapete inhoud (ook) veel auteursrechtelijk beschermde werken bevinden, zoals bijvoorbeeld e-books en afbeeldingen. De gebruikte auteursrechtelijke werken kunnen soms bijvoorbeeld uit illegale bron verkregen zijn, en ook los daarvan zijn rechthebbenden veelal niet op de hoogte van het feit dat hun auteursrechtelijke werken voor de ontwikkeling van een algoritme of AI gebruikt worden.
+
+- Onder auteursrechtjuristen wordt aangenomen dat het gebruik van auteursrechtelijke beschermde werken ter training van algoritmes (waarschijnlijk) een verveelvoudigingshandeling is die de rechthebbende kan verbieden. Dat betekent dat aanbieders van algoritmes het gebruik van auteursrechtelijk beschermd materiaal in de inputfase steeds zullen moeten kunnen legitimeren op grond van (a) toestemming van de rechthebbende(n) of (b) een in de wet neergelegde exceptie op het auteursrechtelijke verveelvoudigingsrecht.
```

Comment:
moest deze zelf even opzoeken, dus iets simpeler proberen op te schrijven. Bart, check jij of het nog steeds klopt hiermee?

```suggestion
- Onder auteursrechtjuristen wordt aangenomen dat het gebruik van auteursrechtelijk beschermde werken ter training van algoritmes (waarschijnlijk) als kopiëren geldt: een handeling die de rechthebbende kan verbieden. Dat betekent dat aanbieders van algoritmes het gebruik van auteursrechtelijk beschermd materiaal in de inputfase steeds moeten kunnen legitimeren op grond van (a) toestemming van de rechthebbende(n) of (b) een in de wet neergelegde exceptie op het auteursrechtelijke verveelvoudigingsrecht.
```